python/pyspark/ml/regression.py

```diff
@@ -791,19 +792,34 @@ class DecisionTreeRegressor(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredi
     >>> model.transform(test1).head().variance
     0.0
 
+    >>> df3 = spark.createDataFrame([
```

Comment:
I guess I mean should we maintain an example without weights too? that might be the more common case. They don't both have to print depth, numNodes, etc.

Reply:
There already exists an example without weights. In the PR, I added examples with weights.
I think it is fine to keep those both. And I am OK to remove some info printed in the added ones.